docs/guides/guide-for-latex-users.md

```diff
@@ -562,6 +562,11 @@ The example below
 This should be a good starting point! If you want to go further, why not create
 a reusable template?
 
+## Bibliographies
+Typst uses [CSL](https://github.com/citation-style-language/styles)
+for citations. This may lead to some differences in citation styles compared to
+LaTeX. But the Typst variation should be the correct one.
```

Comment:
I think that the guide should focus on the benefit the user gets instead of the technologies Typst uses to obtain these benefits. Here are a few benefits:

- Over 80 included citation styles
- Compatibility with many more CSL styles (link to repo)
- Bring your existing BibTeX files

The tone of this paragraph should not be too advertis-y and instead help the user to see whether Typst can meet their requirements and how they can transition. I would not make a note that Typst's output is necessarily more correct than BibLaTeX's.

Reply:
Is this okay the way it's in the latest commit?